### PR TITLE
LPS-125550 rebase on LPS-128599

### PR DESCRIPTION
**Pull request by yuhai**
@tinatian, I add the last two commits for you review clearly. And do the change about the commit https://github.com/tinatian/liferay-portal/commit/2a4259f539b3a1228d881b3d43549f65118ccca0 (because the pr rebased on https://github.com/shuyangzhou/liferay-portal/pull/9823). In addition, after LPS-128599 merged to master. The pr should lack packageinfo change due to the commit https://github.com/tinatian/liferay-portal/commit/2a4259f539b3a1228d881b3d43549f65118ccca0 (I add new constructor for CaptureAppender and LogEvent).

cc @dantewang

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

File: modules/core/portal-bootstrap/system.packages.extra.bnd [hunk 1]
```diff
@@ -223,6 +223,12 @@ Export-Package:\
 	org.apache.commons.logging.impl;version="1.2",\
 	org.apache.commons.validator.routines;version='1.5.1',\
 	org.apache.log4j;version='2.14.0',\
+	org.apache.log4j.bridge;version='2.14.0',\
```
Comment (dantewang) [inline modules/core/portal-bootstrap/system.packages.extra.bnd]: @yuhai portal-test is deployed as a module, but my `RollingFileAppenderBuilder` is in petra-log4j which is deployed as a lib in core, which is in the same class loader as log4j jars. What packages does the latter one need?

Comment (yuhai) [reply modules/core/portal-bootstrap/system.packages.extra.bnd]: @dantewang, except org.apache.log4j.spi, RollingFileAppenderBuilder needs the other packages. I will remove these.

Comment (yuhai) [reply modules/core/portal-bootstrap/system.packages.extra.bnd]: @dantewang, I just push force the change. Only keep portal-test needed.

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by yichenroy** [comment]
@yuhai @tinatian should the following pull request go to `liferay-plugins-ee` instead of `liferay-plugins`?

https://github.com/shuyangzhou/liferay-plugins/pull/162

**Pull request closed without merging**